src/app/app.module.ts

```diff
@@ -53,6 +53,17 @@ export class AppModule {
         }),
         cache: new InMemoryCache(),
       },
-      'chat');
+      'chat'
+    );
+
+    this.apollo.create(
```

Comment:
reason to use this way. in apollo version 2. this is the way we need to link graphQL server. we are still using apollo version 1. when we updating to version 2 this will be  fine.